make/autoconf/jdk-options.m4

```diff
@@ -589,7 +589,7 @@ AC_DEFUN([JDKOPT_ENABLE_DISABLE_COMPATIBLE_CDS_ALIGNMENT],
   UTIL_ARG_ENABLE(NAME: compatible-cds-alignment, DEFAULT: $COMPATIBLE_CDS_ALIGNMENT_DEFAULT,
       RESULT: ENABLE_COMPATIBLE_CDS_ALIGNMENT,
       DESC: [enable use alternative compatible cds core region alignment],
-      DEFAULT_DESC: [disabled],
+      DEFAULT_DESC: [disabled except on linux-aarch64],
```

Comment:
This should not change: for OpenJ9 it's disabled on all platforms.
Line 213 should be deleted (it was removed upstream by ffe232851760239d217534f106652bbf2f195a19).

Reply:
Restored OpenJ9 change, and also match upstream deletion.